dashboard/state_aggregator.py

```diff
@@ -648,7 +648,9 @@ async def summarize_tasks(self, option: SummaryApiOptions) -> SummaryApiResponse
         # For summary, try getting as many entries as possible to minimze data loss.
         result = await self.list_tasks(
             option=ListApiOptions(
-                timeout=option.timeout, limit=RAY_MAX_LIMIT_FROM_API_SERVER, filters=[]
```

Comment:
can we have a unit test in `test_state_api`? maybe we can just have a very simple test to verify job id filter works for ray summary tasks.